src/utils/parseError.ts

```diff
@@ -158,12 +163,21 @@ export const parseError = async (
 
   switch (e.code) {
     case 'CALL_EXCEPTION':
+      if (e.reason === ERC20AllowanceError) {
```

Comment:
Can we use something like the above `e.reason?.includes('...')` here instead of creating a const just for this case? Also, we can consider refactoring those places above. We need to choose one approach for consistency 🙂

Reply:
@Abhikumar98 Sorry, I'm afraid you missed the point here. I was referring to using [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) as a function there, not a property to compare with. 🙂

Reply:
Similar to this https://github.com/lifinance/sdk/pull/127/files#diff-4233cef87e84b84c5873476b8d547421036fa6283d27cb907461b9bdbd1a6af1R120-R121